src/components/table.js

```diff
@@ -394,25 +394,35 @@ GridTable.propTypes = {
   onScroll: PropTypes.func
 }
 
-export const SimpleTable = ({ columns, rows }) => {
+export const SimpleTable = ({ columns, rows, ...props }) => {
   const cellStyles = { paddingTop: '0.25rem', paddingBottom: '0.25rem' }
   return h(Fragment, [
-    div({ style: { display: 'flex' } }, [
+    div({
+      style: { display: 'flex' },
+      role: 'grid',
```

Comment:
Adds `grid`, `row`, `columnheader` and `gridcell` roles to indicate to screen readers that this is a table even though it's marked up with `<div>` tags.

`grid` elements are also supposed to support navigating through each cell using the arrow keys, but that's for later...